doc/source/release/1.12.0-notes.rst

```diff
@@ -33,73 +47,243 @@ New features
--------------------------
-
-
-Other
------
-
+- Added `scipy.stats.quantile_test`, a nonparametric test of whether a
+  hypothesized value is the quantile associated with a specified probability.
+  The ``confidence_interval`` method of the result object gives a confidence
+  interval of the quantile.
+- `scipy.stats.wasserstein_distance` now computes the Wasserstein distance
+  in the multidimensional case.
+- `scipy.stats.sampling.FastGeneratorInversion` provides a convenient
+  interface to fast random sampling via numerical inversion of distribution
+  CDFs.
+- `scipy.stats.geometric_discrepancy` adds geometric/topological discrepancy
+  metrics for random samples.
+- `scipy.stats.multivariate_normal` now has a ``fit`` method for fitting
+  distribution parameters to data via maximum likelihood estimation.
+- `scipy.stats.bws_test` performs the Baumgartner-Weiss-Schindler test of
+  whether two-samples were drawn from the same distribution.
+- `scipy.stats.jf_skew_t` implements the Jones and Faddy skew-t distribution.
+- `scipy.stats.anderson_ksamp` now supports a permutation version of the test
+  using the ``method`` parameter.
+- The ``fit`` methods of `scipy.stats.halcauchy`, `scipy.stats.halflogistic`, and
+  `scipy.stats.halfnorm` are faster and more accurate.
+- `scipy.stats.beta` ``entropy`` accuracy has been improved for extreme values of
+  distribution parameters.
+- The accuracy of ``sf`` and/or ``isf`` methods have been improved for
+  several distributions: `scipy.stats.burr`, `scipy.stats.hypsecant`,
+  `scipy.stats.kappa3`, `scipy.stats.loglaplace`, `scipy.stats.lognorm`,
+  `scipy.stats.lomax`, `scipy.stats.pearson3`, `scipy.stats.rdist`, and
+  `scipy.stats.pareto`.
+- The following functions now support parameters ``axis``, ``nan_policy``, and ``keep_dims``: `scipy.stats.entropy`, `scipy.stats.differential_entropy`, `scipy.stats.variation`, `scipy.stats.ansari`, `scipy.stats.bartlett`, `scipy.stats.levene`, `scipy.stats.fligner`, `scipy.stats.cirmean, `scipy.stats.circvar`, `scipy.stats.circstd`, `scipy.stats.tmean`, `scipy.stats.tvar`, `scipy.stats.tstd`, `scipy.stats.tmin`, `scipy.stats.tmax`, and `scipy.stats.tsem`.
+- The ``logpdf`` and ``fit`` methods of `scipy.stats.skewnorm` have been improved.
+- The beta negative binomial distribution is implemented as `scipy.stats.betanbinom`.
+- The speed of `scipy.stats.invwishart` ``rvs`` and ``logpdf`` have been improved.
+- A source of intermediate overflow in `scipy.stats.boxcox_normmax` with ``method='mle'`` has been eliminated, and the returned value of ``lmbda`` is constrained such that the transformed data will not overflow.
+- `scipy.stats.nakagami` ``stats`` is more accurate and reliable.
+- A source of intermediate overflow in `scipy.norminvgauss.pdf` has been eliminated.
+- Added support for masked arrays to ``stats.circmean``, ``stats.circvar``,
+  ``stats.circstd``, and ``stats.entropy``.
+- ``gausshyper`` ``cdf`` and ``rvs`` methods have improved performance.
```

Comment:
```suggestion
```

The PR https://github.com/scipy/scipy/pull/18799 this refers to  is difficult to decipher: in the end it was basically a simplifcation of a formula with no notable impact for users.